pwnlib/commandline/libcdb.py

```diff
@@ -177,8 +177,11 @@ def translate_offset(offs, args, exe):
 
 def collect_synthetic_symbols(exe):
     available_symbols = ['str_bin_sh']
-    exe.symbols['str_bin_sh'] = next(exe.search(b'/bin/sh\x00'))
-
+    try:
+        exe.symbols['str_bin_sh'] = next(exe.search(b'/bin/sh\x00'))
+    except:
```

Comment:
Please specify the actually intended exception here, I believe it is `StopIteration` (or was it `IndexError`, not sure). See [the linter rule rationale](https://www.flake8rules.com/rules/E722.html).